addon/components/polaris-badge.js

```diff
@@ -65,44 +60,43 @@ export default class PolarisBadge extends Component {
    * @default null
    * @public
    */
-  progress = null;
+  progress;
 
   /**
    * Medium or small size. Use `small` only in the main navigation of an app frame..
```

Comment:
Well this bugs me (so this is probably where @tomnez adds 7 more `.`s at the end 😂)

```suggestion
   * Medium or small size. Use `small` only in the main navigation of an app frame.
```